### PR TITLE
Add transforms: RandomErasing and TrivialAugmentationWide

### PR DESCRIPTION
**Pull request by illian01**
## Description

Please include a summary in English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #240

You should link at least one existing issue for PR. Before your create a PR, please check to see if there is an issue for this change.  
PRs from forked repository not accepted.

## Change(s)

- Add transform functions `RandomErasing` and `TrivialAugmentationWide`

## Changelog

If you PR to `dev` branch, please add a brief summary of the change to the **Upcoming Release** section of the [`CHANGELOG.md`](https://github.com/Nota-NetsPresso/netspresso-trainer/blob/master/CHANGELOG.md) file and include a link to the PR (formatted in markdown) and a link to your github profile.

For example,

```
- Added a new feature by `@myusername` in [PR 2023](https://github.com/Nota-NetsPresso/netspresso-trainer/pull/2023)
```

## Code Formatting

If you PR to either `master` or `dev` branch, you should follow the code linting process. Please check your code with `lint_check.sh` in `./scripts` directory.
For more information, please read the contribution guide in `CONTRIBUTING.md`.

### CODE REVIEW

File: src/netspresso_trainer/dataloaders/augmentation/custom.py [hunk 1]
```diff
@@ -351,6 +354,130 @@ def __repr__(self):
         return format_string
 
 
+class RandomErasing(T.RandomErasing):
+
```
Comment (deepkyu) [inline src/netspresso_trainer/dataloaders/augmentation/custom.py:358]: class variable `visualize`

If not yet implemented for visualize, please set this as False .

File: src/netspresso_trainer/dataloaders/augmentation/custom.py [hunk 2]
```diff
@@ -351,6 +354,130 @@ def __repr__(self):
...
+            return image, mask, bbox
+        return image, mask, bbox
+
+
+class TrivialAugmentWide(torch.nn.Module):
+    """
+    Based on the torchvision implementation.
+    https://pytorch.org/vision/main/_modules/torchvision/transforms/autoaugment.html#TrivialAugmentWide
+    """
+
```
Comment (deepkyu) [inline src/netspresso_trainer/dataloaders/augmentation/custom.py:413]: Same here

**Review by deepkyu — COMMENTED** [review]
Thanks for the update. Check the comment plz

**Comment by deepkyu** [comment]
TODO for later:
- Apply mask and bbox
  - For random erasing, this could be extended to object-aware method.

**Review by deepkyu — APPROVED** [review]
LGTM
